manifests/init.pp

```diff
@@ -61,6 +61,9 @@
 #
 # $qpid_router_logging_path::           Directory for dispatch router logs
 #
+# $docker_port::                        Docker registry port, 5000 (default) or 5001
```

Comment:
I like the more descriptive name; will change.

I can leave off the values but if this message is shown to users (is it?) then only 5000 and 5001 will work with selinux. Let me know to leave, remove, or change verbage.